src/main/pg/flash.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#ifdef USE_FLASHFS
+
+#include "pg/pg.h"
+#include "pg/pg_ids.h"
+
+#include "drivers/bus_spi.h"
+#include "drivers/io.h"
```

Comment:
`pg` after `drivers`

Reply:
Erg, I didn't even touch that (besides copying the file). That's just a remnant of moving these files from `config` to `pg` with a search / replace.